akd/src/storage/memory.rs

```diff
@@ -87,6 +87,13 @@ impl Storage for AsyncInMemoryDatabase {
     async fn commit_transaction(&self) -> Result<(), StorageError> {
         // this retrieves all the trans operations, and "de-activates" the transaction flag
         let ops = self.trans.commit_transaction().await?;
+        let _epoch = match ops.last() {
+            Some(DbRecord::Azks(azks)) => Ok(azks.latest_epoch),
```

Comment:
For my own understanding, can you expand on why checking the last item in the `Vec` is appropriate in these checks for determining if the kind of `DbRecord` is `Azks`?

Reply:
We need to make sure that the last thing committed is the Azks record as that is what triggers concurrent reading entities that a new epoch has occurred. The transaction is applying a sorting when it's generating the transaction set, and this just asserts that the sort was correct and the last element is the right one to trigger the notification to everything that there's something new.